xray/formats/conversion.go

```diff
@@ -81,6 +148,16 @@ func ConvertToComponentTableRow(rows []ComponentRow) (tableRows []DirectDependen
 	return
 }
 
+func ConvertToComponentScanTableRow(rows []ComponentRow) (tableRows []DirectPackagesTableRow) {
```

Comment:
It looks like this function can be made private.

Reply:
Done